src/piadm.sh

```diff
@@ -724,30 +719,31 @@ install_pi_CN() {
 	# For now, use bootparams to get the URL needed, and pull
 	# files from there.  If there's a better way to obtain things, use it.
 	unix_path=$(bootparams | grep boot-file | awk -F= '{print $2}')
-	archive_prefix=$(echo $unix_path | sed 's/kernel\/amd64\/unix/amd64/g')
+	archive_prefix=$(echo "$unix_path" | sed 's/kernel\/amd64\/unix/amd64/g')
```

Comment:
Cleaner (but not a big deal):

    archive_prefix=$(sed 's#kernel/amd64/unix#amd64#g' <<< "$unix_path")

Reply:
Will hold off on this one.

Reply:
Actually, since I have the patient open again for @papertigers , I'll fix these two as well.

Reply:
And now that I see it...I have to 1-up @papertigers here.

    unix_path=$(bootparams | awk -F= ‘/^boot-file=/ {print $2}’

I.e., avoid `if` if a regexp works.

Reply:
Syntax error?

```
[root@larry (kebecloud) ~]# unix_path=$(bootparams | awk -F= ‘/^boot-file=/ {print $2}’)
awk: syntax error at source line 1
 context is
	 >>> ? <<< 
awk: bailing out at source line 1
[root@larry (kebecloud) ~]# 
```

Reply:
Old comment deleted because tick vs. backtick was the problem.